lib/OpenQA/Worker/Jobs.pm

```diff
@@ -153,6 +154,16 @@ sub upload {
     close $log;
     printf "uploading %s\n", $filename if $verbose;
 
+    my $retry_counter;
+    my $retry_limit = 5;
+    my $tics        = 5;
+
+  START_UPLOAD:
+    if ($retry_counter) {
+        printf "WARNING: Upload retry #%s/%s of %s in 5 seconds: ", $retry_counter, $retry_limit, $filename;
```

Comment:
I will do it in a later PR, there are few things that need to be refactored at least here (And possibly somewhere else), log calls is one of them